modules/azurerm-network-security-group/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "this" {
+  description = "Azure Network Security Rule."
```

Comment:
small nitpick, but would it be more descriptive to say "... Security Group and Rules"?

Reply:
I tried to make the module in a way that would support both so I think you could go either way here, but considering (atleast so far) we're using it for in-line resources, I think it's a good point.